### ISSUE TITLE
Can't install with SQLite

### ISSUE THREAD

**Issue by peter-kehl** [thorsten/phpMyFAQ#1194]
When running on with PHP that only has SQLite support (but no support for other DB types supported by phpMyFAQ), SQLite3 shows up on the installer list. However, field 'sql_sqlitefile' stays hidden and therefore I can't fill it in. I put the SQLite file path to "Host", but that is ignored. After submitting the installer freezes.

Guessing the bug: From memory, the installer used to show/hide SQLite-specific field, once the user selected SQLite3 from the list. However, since I have only SQLite3 on the list, it's pre-selected. Therefore I can't select it (again), hence no change event, and therefore Javascript doesn't show the hidden field.

This is with both git master and 2.10.0.
```
<div id="dbsqlite" class="hide">
                        <div class="form-group">
                            <label class="col-sm-4 control-label" for="sql_sqlitefile">SQLite file:</label>
                            <div class="col-sm-8">
                                <input type="text" name="sql_sqlitefile" id="sql_sqlitefile"
                                       value="/var/www/html/phpMyFAQ/phpmyfaq" class="form-control">
                                <p class="help-block">
                                    Please enter the full path to your SQLite datafile which should be outside your
                                    docroot.
                                </p>
                            </div>
                        </div>
                    </div>

Following logs in apache error log:
Exception' with message 'Unable to open database: unable to open database file' in /var/www/html/phpMyFAQ/phpmyfaq/inc/PMF/DB/Sqlite3.php:75\nStack trace:\n#0 /var/www/html/phpMyFAQ/phpmyfaq/inc/PMF/DB/Sqlite3.php(75): SQLite3->__construct('/var/www/html/p...')\n#1 /var/www/html/phpMyFAQ/phpmyfaq/inc/PMF/Installer.php(646): PMF_DB_Sqlite3->connect('/var/www/html/p...', 'dummyUserName', 'dummyPasswordHere', 'dfg')\n#2 /var/www/html/phpMyFAQ/phpmyfaq/setup/index.php(422): PMF_Installer->startInstall()\n#3 {main}\n  thrown in /var/www/html/phpMyFAQ/phpmyfaq/inc/PMF/DB/Sqlite3.php on line 75, referer: http://localhost/phpMyFAQ/phpmyfaq/setup/index.php
```

**Comment by thorsten** [comment]
Did you tried 2.9 as well?

**Comment by peter-kehl** [comment]
I've tested with 2.9. It has the same problem.

I'm suggesting a fix (as per Javascript situation described above): https://github.com/thorsten/phpMyFAQ/pull/1195

**Comment by thorsten** [comment]
Fixed in 2.9.2 and merged into 2.10 branch
